src/adapter/src/coord.rs

```diff
@@ -1303,6 +1313,31 @@ impl<S: Append + 'static> Coordinator<S> {
         self.pending_writes.push(pending_write_txn);
     }
 
+    //  #[tracing::instrument(level = "debug", skip_all)]
+    async fn storage_usage_update(&mut self) {
+        // TODO: call persist API, stub out values for now
+        let object_id = None;
+        let size_bytes = 12345;
+        // let insertion = self
+        //     .catalog
+        //     .state()
+        //     .pack_storage_usage_update(storage_id, bytes_used, timestamp);
+        // let updates = vec![insertion];
+        // self.send_builtin_table_updates(updates).await;
+        // Persist update in stash
+
+        self.catalog_transact(
+            None,
+            vec![catalog::Op::UpdateStorageMetrics {
+                object_id,
+                size_bytes,
+            }],
+            |_| Ok(()),
+        )
+        .await
+        .expect("updating storage usage status cannot fail");
```

Comment:
This expect should be changed to log the error with maybe `tracing::warn!` instead of crash? Or possibly retry somehow? Logging probably fine for now.